High-level-design.md

```diff
@@ -256,7 +256,7 @@ There are currently three types of `CoinJoinerPeer`:
 2. `Taker`
 3. `Maker`
 
-`OrderbookWatch` does not participate in the Joinmarket protocol, but is an abstraction to allow observation of the Joinmarket pit. It stores order information in a sqlite3 db object.
+`OrderbookWatch` does not participate in the Joinmarket protocol, but is an abstraction to allow observation of the Joinmarket pit. It stores order information in an sqlite3 db object.
```

Comment:
The database library's name is pronounced "sequel-lite", so "an sqlite3" is actually wrong.

Reply:
Is it, though?  I was under the impression "Sequel" was a microsoft copyrighted product name and not a pronunciation that generic SQL databases use, but I could be very wrong.  Happy to change this one back if that's the case.

Reply:
Nobody gives a shit about my code review, so wait for one of the benevolent bureaucrats for slightly more officious opinions.